apps/intellij/src/main/kotlin/dev/nx/console/run/runGeneralCommand.kt

```diff
@@ -0,0 +1,64 @@
+package dev.nx.console.run
+
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.executors.DefaultRunExecutor
+import com.intellij.execution.filters.TextConsoleBuilderFactory
+import com.intellij.execution.process.KillableColoredProcessHandler
+import com.intellij.execution.ui.RunContentDescriptor
+import com.intellij.execution.ui.RunContentManager
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import dev.nx.console.NxIcons
+import dev.nx.console.nxls.NxlsService
+import dev.nx.console.utils.ProjectLevelCoroutineHolderService
+import dev.nx.console.utils.nxBasePath
+import java.io.File
+import javax.swing.SwingUtilities
+import kotlinx.coroutines.launch
+
+fun runGeneralCommand(project: Project, projectName: String, helpCommand: String) {
```

Comment:
This doesn't really seem like a `runGeneralCommand` implemention but more like a `runHelpCommand` implementation right? I think variable & function naming should be made consistent either way.

Reply:
Yeah the intention was originally running any command, but it is more specific now.